src/internal.c

```diff
@@ -9468,11 +9449,14 @@ static int DoVerifyCallback(WOLFSSL* ssl, int ret, ProcPeerCertArgs* args)
     }
 #endif
     /* if verify callback has been set */
-    if (use_cb && (ssl->verifyCallback
+    if (use_cb && (((ssl != NULL) && (ssl->verifyCallback != NULL))
```

Comment:
Let's clean this up... Logic is: `((use_cb && (ssl && (ssl-> verifyCallback || ssl->ctx->verifyCertCb))) || (cm && cm->verifyCallback))`